liquibase-service/src/main/resources/db/odse/routines/021-sp_interview_event-001.sql

```diff
@@ -0,0 +1,1291 @@
+                        )                                              AS COMMENT_DATE
+        INTO #INTERVIEW_NOTE
+        FROM #INTERVIEW_NOTE_INIT
+
+        if
+            @debug = 'true'
+            select @Proc_Step_Name as step, *
+            from #INTERVIEW_NOTE;
+
+        SELECT @RowCount_no = @@ROWCOUNT;
+
+        INSERT INTO [rdb_modern].[dbo].[job_flow_log]
+        (batch_id, [Dataflow_Name], [package_Name], [Status_Type], [step_number], [step_name], [row_count])
+        VALUES (@batch_id, 'Interview PRE-Processing Event', 'nrt_interview', 'START', @Proc_Step_no, @Proc_Step_Name,
+                @RowCount_no);
+
+
+        COMMIT TRANSACTION;
+
+        BEGIN
+            TRANSACTION;
+        SET
+            @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET
+            @PROC_STEP_NAME = 'GENERATING #D_INTERVIEW_COLUMNS';
+
+        WITH ordered_list AS (SELECT 'D_INTERVIEW'                                                              AS TABLE_NAME,
+                                     RDB_COLUMN_NM,
+                                     1                                                                          AS NEW_FLAG,
+                                     LAST_CHG_TIME,
+                                     LAST_CHG_USER_ID,
+                                     ROW_NUMBER() OVER (PARTITION BY RDB_COLUMN_NM ORDER BY LAST_CHG_TIME DESC) AS rn
+
+                              FROM NBS_ODSE.dbo.NBS_rdb_metadata
+                              WHERE RDB_TABLE_NM = 'D_INTERVIEW'
+                                AND RDB_COLUMN_NM NOT IN (
+                                                          'IX_STATUS',
+                                                          'IX_TYPE',
+                                                          'IX_INTERVIEWEE_ROLE',
+                                                          'IX_DATE',
+                                                          'IX_LOCATION'
+                                  ))
+        SELECT TABLE_NAME,
+               RDB_COLUMN_NM,
+               NEW_FLAG,
+               LAST_CHG_TIME,
+               LAST_CHG_USER_ID
+        INTO #D_INTERVIEW_COLUMNS
+        FROM ordered_list
+        where rn = 1
+
+
+        if
+            @debug = 'true'
+            select @Proc_Step_Name as step, *
+            from #D_INTERVIEW_COLUMNS;
+
+        SELECT @RowCount_no = @@ROWCOUNT;
+
+        INSERT INTO [rdb_modern].[dbo].[job_flow_log]
+        (batch_id, [Dataflow_Name], [package_Name], [Status_Type], [step_number], [step_name], [row_count])
+        VALUES (@batch_id, 'Interview PRE-Processing Event', 'nrt_interview', 'START', @Proc_Step_no, @Proc_Step_Name,
+                @RowCount_no);
+
+
+        COMMIT TRANSACTION;
+
+
+        BEGIN
+            TRANSACTION;
+        SET
+            @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET
+            @PROC_STEP_NAME = 'SELECT FULL INTERVIEW DATA';
+
+        SELECT ix.INTERVIEW_UID,
```

Comment:
💯 Massive migration 
What is the average run time time seen for this sproc?

Reply:
For all of the Interviews (~50000) in Alabama, it takes about 8 seconds to aggregate all of the Interview UIDs (necessary to create the input string for the list of uids that the sproc needs) and then push all of the results to a temp table instead of displaying them on my PC, which takes a bit of time. Any individual interview or small sets of interviews have been <0.5 seconds to process and display the results